contracts/src/BeefyClient.sol

```diff
@@ -215,6 +216,9 @@ contract BeefyClient {
         ValidatorSet memory _initialValidatorSet,
         ValidatorSet memory _nextValidatorSet
     ) {
+        if (_nextValidatorSet.id != _initialValidatorSet.id + 1) {
+            revert InvalidValidatorSetData();
```

Comment:
Can just `revert()` here without raising a custom exception. I don't want to add a custom exception `InvalidValidatorSetData` which may never ever be used.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1025/commits/9fd119560d062741cc7513e102778fa7d21e5850.

Reply:
Nit, would prefer a string that looks more robotic: `invalid-constructor-params`